src/Adapter/Presenter/Order/OrderLazyArray.php

```diff
@@ -165,14 +165,16 @@ public function getProducts()
         $order = $this->order;
         $cart = new Cart($order->id_cart);
 
-        $orderProducts = $order->getCartProducts();
+        $orderProducts = $order->getProducts();
         $cartProducts = $this->cartPresenter->present($cart);
         $orderPaid = $order->getCurrentOrderState() && $order->getCurrentOrderState()->paid;
 
         $includeTaxes = $this->includeTaxes();
         foreach ($orderProducts as &$orderProduct) {
             $orderProduct['name'] = $orderProduct['product_name'];
             $orderProduct['quantity'] = $orderProduct['product_quantity'];
+            $orderProduct['id_product'] = $orderProduct['product_id'];
```

Comment:
Could you add a comment here please? Explaining that the replaced data actually comes from the OrderDetail object/table, in case the product row has been removed (hence the difference in array field names)